lib/publiccloud/provider.pm

```diff
@@ -28,8 +29,6 @@ has username => undef;
 has prefix => 'openqa';
 has terraform_env_prepared => 0;
 has terraform_applied => 0;
```

Comment:
maybe we need to add : 
`has vault => undef`  ?

Reply:
done